--- etc/init.d/haproxy-orig	2024-05-21 09:40:51.000000000 +0000
+++ etc/init.d/haproxy	2025-01-07 11:45:01.505035548 +0000
@@ -6,15 +6,18 @@

 USE_PROCD=1

-HAPROXY_BIN="/usr/sbin/haproxy"
-HAPROXY_CONFIG="/etc/haproxy.cfg"
+HAPROXY_DIR='/etc/haproxy'
+HAPROXY_BIN='/usr/sbin/haproxy'
+HAPROXY_CONFIG="${HAPROXY_DIR}/haproxy.cfg"

 start_service() {
 	procd_open_instance
 	procd_set_param respawn
-	procd_set_param file "$HAPROXY_CONFIG"
+	procd_set_param file "${HAPROXY_CONFIG}"
+	procd_set_param stdout 1
+	procd_set_param stderr 1
 	procd_set_param reload_signal USR2
-	procd_set_param command $HAPROXY_BIN -q -W -db -f "$HAPROXY_CONFIG"
+	procd_set_param command bash -c "cd ${HAPROXY_DIR} && exec ${HAPROXY_BIN} -q -W -db -f ${HAPROXY_CONFIG} 2>&1"
 	procd_close_instance
 }

@@ -24,5 +27,5 @@

 extra_command "check" "Check haproxy config"
 check() {
-	$HAPROXY_BIN -c -q -V -f $HAPROXY_CONFIG
+	${HAPROXY_BIN} -c -q -V -f ${HAPROXY_CONFIG}
 }
