--- etc/init.d/dnscrypt-proxy-orig 2024-09-23 12:34:46.000000000 +0000
+++ etc/init.d/dnscrypt-proxy 2024-12-19 06:39:41.602141031 +0000
@@ -16,6 +16,9 @@
         procd_set_param file "$CONFIGFILE"
         procd_set_param stdout 1
         procd_set_param stderr 1
-        procd_set_param respawn ${respawn_threshold:-3600} ${respawn_timeout:-5} ${respawn_retry:-5}
+        procd_set_param respawn \
+          ${respawn_threshold:-333} \
+          ${respawn_timeout:-3} \
+          ${respawn_retry:-0}
         procd_close_instance
 }

--- etc/init.d/stubby-orig 2024-09-23 12:34:46.000000000 +0000
+++ etc/init.d/stubby 2024-12-19 06:38:54.961736954 +0000
@@ -248,7 +248,10 @@
         if [ -n "$command_line_arguments" ]; then
             procd_append_param command "$command_line_arguments"
         fi
-        procd_set_param respawn
+        procd_set_param respawn \
+          ${respawn_threshold:-333} \
+          ${respawn_timeout:-3} \
+          ${respawn_retry:-0}
         procd_set_param file "$stubby_config"
         procd_set_param stdout 1
         procd_set_param stderr 1

--- etc/init.d/uwsgi-orig	2024-09-23 12:34:46.000000000 +0000
+++ etc/init.d/uwsgi	2025-01-03 12:55:40.000000000 +0000
@@ -7,7 +7,11 @@
 	procd_open_instance
 	procd_set_param command /usr/sbin/uwsgi --ini /etc/uwsgi/emperor.ini
 	procd_set_param stderr 1
+	procd_set_param stdout 1
 	procd_set_param file /etc/uwsgi/emperor.ini
-	procd_set_param respawn
+	procd_set_param respawn \
+		${respawn_threshold:-333} \
+		${respawn_timeout:-3} \
+		${respawn_retry:-0}
 	procd_close_instance
 }

--- etc/init.d/collectd-orig	2024-09-23 12:34:46.000000000 +0000
+++ etc/init.d/collectd	2025-03-29 05:57:14.000000000 +0000
@@ -597,7 +597,10 @@
 	procd_append_param command -f # don't daemonize
 	procd_set_param nice "$NICEPRIO"
 	procd_set_param stderr 1
-	procd_set_param respawn
+	procd_set_param respawn \
+		${respawn_threshold:-333} \
+		${respawn_timeout:-3} \
+		${respawn_retry:-0}
 	procd_close_instance
 }

--- etc/init.d/fail2ban-orig	2024-09-23 12:34:46.000000000 +0000
+++ etc/init.d/fail2ban	2025-03-29 05:59:11.000000000 +0000
@@ -31,6 +31,7 @@
 	init_config

 	mkdir -m 0755 -p "${RUNDIR}"
+	ln -sfv /opt/var/log/messages /var/log/messages

 	procd_open_instance

@@ -46,8 +47,11 @@
 	procd_set_param file ${CONFDIR}/*.local

 	procd_set_param command /usr/bin/fail2ban-server -xf -p "${RUNDIR}/fail2ban.pid" -s "${RUNDIR}/fail2ban.sock" start
-	procd_set_param respawn ${respawn_threshold:-3600} ${respawn_timeout:-5} ${respawn_retry:-5}
-	procd_close_instance
+	procd_set_param respawn \
+		${respawn_threshold:-333} \
+		${respawn_timeout:-3} \
+		${respawn_retry:-0}
+	ocd_close_instance
 }

 reload_service()

--- etc/init.d/librespeed-go-orig	2025-10-21 08:16:22.456535933 +0000
+++ etc/init.d/librespeed-go	2025-10-21 08:20:33.242249346 +0000
@@ -68,9 +68,12 @@

 	procd_set_param limits core="unlimited"
 	procd_set_param limits nofile="1000000 1000000"
-	procd_set_param respawn
 	procd_set_param stderr 1
 	procd_set_param user librespeed
+	procd_set_param respawn \
+		${respawn_threshold:-333} \
+		${respawn_timeout:-3} \
+		${respawn_retry:-0}

 	procd_add_jail "$CONF" log
 	procd_add_jail_mount "$TMPCONF"

--- etc/init.d/rsyslog-orig	2025-10-28 12:25:45.265256293 +0000
+++ etc/init.d/rsyslog	2025-10-28 12:29:59.000000000 +0000
@@ -103,6 +103,10 @@
 start_service() {
 	expand_config
 	procd_open_instance
+	procd_set_param respawn \
+		${respawn_threshold:-333} \
+		${respawn_timeout:-3} \
+		${respawn_retry:-0}
 	procd_set_param command /usr/sbin/rsyslogd -f ${CONFIG_FILE} -n
 	procd_close_instance
 }
