--- etc/init.d/uwsgi-orig	2024-09-23 12:34:46.000000000 +0000
+++ etc/init.d/uwsgi	2025-01-03 12:55:40.000000000 +0000
@@ -7,7 +7,11 @@
 	procd_open_instance
 	procd_set_param command /usr/sbin/uwsgi --ini /etc/uwsgi/emperor.ini
 	procd_set_param stderr 1
+	procd_set_param stdout 1
 	procd_set_param file /etc/uwsgi/emperor.ini
-	procd_set_param respawn
+	procd_set_param respawn \
+		${respawn_threshold:-333} \
+		${respawn_timeout:-3} \
+		${respawn_retry:-0}
 	procd_close_instance
 }
